packages/devtools_shared/test/utils/serialization_test.dart

```diff
@@ -6,15 +6,36 @@ import 'package:devtools_shared/src/utils/serialization.dart';
 import 'package:test/test.dart';
 
 void main() {
-  test('deserialize works for json', () {
-    final json = {'key': 'value'};
-    String deserializer(Map<String, dynamic> json) => 'correct';
-    expect(deserialize<String>(json, deserializer), 'correct');
+  group('deserialize', () {
+    test('works for json', () {
+      final json = {'key': 'value'};
+      String deserializer(Map<String, dynamic> json) => 'correct';
+      expect(deserialize<String>(json, deserializer), 'correct');
+    });
+
+    test('works for object', () {
+      const json = 'correct';
+      String deserializer(Map<String, dynamic> json) => 'wrong';
+      expect(deserialize<String>(json, deserializer), 'correct');
+    });
   });
 
-  test('deserialize works for object', () {
-    const json = 'correct';
-    String deserializer(Map<String, dynamic> json) => 'wrong';
-    expect(deserialize<String>(json, deserializer), 'correct');
+  group('deserialize', () {
```

Comment:
fix group and test names here. I think this needs to be "deserializeNullable" and the test cases need "deserializeNullable" removed